pkg/minikube/constants/constants.go

```diff
@@ -234,7 +234,7 @@ func GetKubeadmCachedImages(kubernetesVersionStr string) []string {
 		"k8s.gcr.io/kube-apiserver-amd64:" + kubernetesVersionStr,
 	}
 
-	ge_v1_14 := semver.MustParseRange(">=1.14.0")
+	v1_14 := semver.MustParseRange(">=1.14.0")
```

Comment:
I feel like this name isn't as meaningful any more, but understand why lint picked this up.

Perhaps "newerThan114" or "greaterThan114". "ge114" is probably a valid token too.

Alternatively, give this a max range?

Reply:
I improvised. Most likely this whole section should be rewritten anyway ?